code/datums/communications.dm

```diff
@@ -125,12 +125,14 @@ datum/radio_frequency
 								device.receive_signal(signal, TRANSMISSION_RADIO, frequency)
 					else
 						device.receive_signal(signal, TRANSMISSION_RADIO, frequency)
+					LAGCHECK(LAG_REALTIME)
 
 			if (!reusable_signals || reusable_signals.len > 10)
 				signal.dispose()
 			else if (signal)
 				signal.wipe()
 				reusable_signals |= signal
+			LAGCHECK(LAG_MED)
```

Comment:
does this one actually do anything? It's at the end of the proc not in a loop

Reply:
Probably not. MBC did say he added them liberally when he first did it long ago.